### ISSUE TITLE
Custom WroManagerFactory

### ISSUE THREAD

**Issue by lwd369** [michael-simons/wro4j-spring-boot-starter#142]
To use custom name strategy I created a WroManagerFactory bean:

``` java 
@Configuration
public class Wro4jConfig {
  @Bean
  WroManagerFactory wroManagerFactory(WroModelFactory modelFactory) {

    ConfigurableWroManagerFactory wroManagerFactory = new ConfigurableWroManagerFactory();
    wroManagerFactory.setNamingStrategy(new DefaultHashEncoderNamingStrategy());
    wroManagerFactory.setModelFactory(modelFactory);

    return wroManagerFactory;
  }

}
```

From [doc](https://github.com/michael-simons/wro4j-spring-boot-starter#not-configurable-at-the-moment) it should inject the modelFactory bean (or others). But WroModelFactory bean is not found when start application.

The WroAutoConfiguration code :

``` java
@Bean
@ConditionalOnMissingBean({WroManagerFactory.class, WroModelFactory.class})
WroModelFactory wroModelFactory(final Wro4jProperties wro4jProperties) {
    return new ConfigurableXmlModelFactory(wro4jProperties.getModel());
}
```

`@ConditionalOnMissingBean({WroManagerFactory.class, WroModelFactory.class})` is that means if there is a WroManagerFactory bean, then nothing will happen?

And how I can create a custom WroManagerFactory to use the name strategy?

**Comment by michael-simons** [comment]
I'm terribly sorry for the super late reply.

Yes, this is a bug.

**Comment by michael-simons** [comment]
Will be fixed with 0.11.4, on Maven central soon.
